controller/src/functions/renderings-bus.ts

```diff
@@ -42,14 +45,50 @@ export async function wacsSbRenderingsApi(
 ) {
   // If zod or the db action below throws here, the message will end up in the dead letter queue.
   try {
+    const successStatus = z.object({Successful: z.boolean()}).parse(message);
+    const wasSuccessful = !!successStatus.Successful;
+    if (!wasSuccessful) {
+      context.warn(
+        `RENDERINGS BUS RECEIVED: received message from bus that was not a successful render. NOOP.  ${JSON.stringify(
+          message
+        )}`
+        //  noop at this point. can't do anything.
+      );
+      return;
+    }
     const parsed = renderingsSchema.parse(message);
+    const namespace = "wacs";
+    const id = `${parsed.User}/${parsed.Repo}`.toLowerCase();
     context.log(
-      `received a message for ${parsed.User} for ${parsed.Repo} for ${parsed.ResourceType} type`
+      `RENDERINGS BUS RECEIVED: received a message for ${parsed.User} for ${parsed.Repo} for ${parsed.ResourceType} type`
     );
 
+    const doesExist = await checkContentExists(`${namespace}-${id}`);
+    if (!doesExist) {
+      context.log(
+        `${namespace}-${id} is not already in api. Creating new row in table`
+      );
+      const newContentRow: z.infer<typeof validators.contentPost> = [
+        {
+          namespace: "wacs",
+          id: `${parsed.User}/${parsed.Repo}`.toLowerCase(),
+          type: "text",
+        },
+      ];
+      const newRowRes = await handleContentPost(newContentRow);
+      if (newRowRes.status !== 200) {
+        context.log(
+          `Failed to create new content row for ${`${parsed.User}/${parsed.Repo}`.toLowerCase()}`
+        );
+        throw new Error(
+          `Failed to create new content row for ${`${parsed.User}/${parsed.Repo}`.toLowerCase()}`
+        );
+      }
+    }
+
```

Comment:
@PurpleGuitar , @danparisd  - Here is the logic for creating a content row in the db if it doesn't exist before proceeding to try to adjust add all the blobs/renders of that content